modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/components/ActionCreationForm.tsx

```diff
@@ -114,13 +122,21 @@ const ActionCreationForm = ({
 			body.label = labelTranslations;
 		}
 
-		const response = await fetch(API_URL.FDS_ACTIONS, {
+		let fetchURL = API_URL.FDS_ACTIONS;
+		let fetchMethod = 'POST';
+
+		if (activeSection === sections.EDIT_ITEM_ACTION) {
+			fetchURL = `${API_URL.FDS_ACTIONS}/${initialValues?.id}`;
+			fetchMethod = 'PUT';
```

Comment:
Is this `PUT` intentional? We could make this a `PATCH` and send only updates, but that would complicate code. This is probably fine.

Reply:
I used `PUT` since it was mentioned in the ticket to use it. 🤷 I do like `PUT` in this sense though since it's editing the entire same form.

https://liferay.atlassian.net/browse/LPS-191488
> Technical notes:
> 
> - Use the PUT endpoint to save all the changes